src/scenes/NftDetailPage/NftDetailText.tsx

```diff
@@ -35,6 +37,14 @@ function NftDetailText({
   const horizontalLayout = breakpoint === size.desktop || breakpoint === size.tablet;
   const addressToUse = creatorAddress || contractAddress || '';
 
+  // useRef to prevent rendered username from briefly changing before fade transition upon route change
+  const username = useRef(ownerUsername);
```

Comment:
ooo interesting! what would it change to?

Reply:
since the username comes from the route, the value changed to whatever the new route was (`edit`, `community`, etc)

when going from gallery.so/kaito -> gallery.so/edit

Reply:
ohhhh right, great call!

damn transitioner lol